### PR TITLE
Update Maintainers List

### PR DESCRIPTION
**Pull request by Foggalong**
Updates the Code Owners list so that @palob also has ownership rights and (should) be able to merge PRs without my approval. Also added an explicit notice about who the "maintainers" are for this repository, will add similar to others.

### CODE REVIEW

**Comment by Foggalong** [comment]
Tangential, but I've given the @numixproject/icons team maintainer rights to these repositories - think that should give everyone more rights to transfer issues between repos, use moderation tools, etc.
   
Screenshot from 2021-04-14 05-24-10

**Review by palob — APPROVED** [review]

**Comment by palob** [comment]
So this is the new situation:

+ I can merge PRs of others if there is no pending disapproving review of yours somewhere up the queue
+ I cannot push directly to master, need to create a PR first which it seems I can only merge after your approval
+ In PRs of others I cannot overrule your old disapproving reviews somewhere up the queue

**Comment by Foggalong** [comment]
Huh that is weird, I'm not sure why you can't overrule my shit or push straight to master because I thought we were on equal footing  now permissions wise. I'll do some digging in the settings and work out what the mismatch is there, thanks for flagging!

**Comment by palob** [comment]
Direct committing to the main branch works! Will check how are things with PRs during the next week.

**Comment by Foggalong** [comment]
Awesome, thanks so much @palob! :heart:

**Comment by palob** [comment]
Can merge PRs from Numix branches, from fork branches and I see the merge button in branches with your pending disapproving reviews :+1:

**Comment by Foggalong** [comment]
Eeeexcellent :heart:

**Pull request merged**
